### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Addresses issue https://github.com/OfficeDev/office-store-docs/issues/186. Also minor cleanup (includes updating links from dev.office.com).

### CODE REVIEW

**Comment by Lauragra** [comment]
OPS Build status updates of commit _[469340d](https://github.com/OfficeDev/office-store-docs/pull/193/commits/469340d3a76b8942d595052462fbbb627228cef1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-license-checks-to-office-and-sharepoint-add-ins.md](https://github.com/OfficeDev/office-store-docs/blob/elizs-fix-links/docs/add-license-checks-to-office-and-sharepoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/promote-your-office-store-solution.md](https://github.com/OfficeDev/office-store-docs/blob/elizs-fix-links/docs/promote-your-office-store-solution.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C12%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201903121606118283-193%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=22c4kP1aqk4SSboi89szlsgEnwIwX6Om8DKS20jsC88%3D&st=2019-03-12T16%3A02%3A13Z&se=2019-04-12T16%3A07%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/promote-your-office-store-solution.md [hunk 1]
```diff
@@ -11,25 +11,25 @@ After you submit your solution to the Seller Dashboard and it is approved for ad
...
    |:-----|:-----|
    |Small|![Screenshot of the small-sized AppSource badge](images/appsource-badge-small.png)|
    |Medium|![Screenshot of the medium-sized AppSource badge](images/appsource-badge-medium.png)|
    |Large|![Screenshot of the large-sized AppSource badge](images/appsource-badge-large.png)|
 
-3. Add the badge to your app or add-in webpage, directly following the link. This makes it clear to your users that they can download your app or add-in from AppSource. 
+3. Add the badge to your app or add-in webpage, directly following the link. This makes it clear to your users that they can download your app or add-in from AppSource.
 
-You can also add your Office 365 web app using Azure AD to the Office 365 app launcher. For information, see [Have your add-in appear in the Office 365 add-in launcher](https://msdn.microsoft.com/en-us/office/office365/howto/connect-your-app-to-o365-app-launcher).
+You can also add your Office 365 web app using Azure AD to the Office 365 app launcher. For information, see [Have your add-in appear in the Office 365 add-in launcher](https://msdn.microsoft.com/office/office365/howto/connect-your-app-to-o365-app-launcher).
```
Comment (Lauragra) [inline docs/promote-your-office-store-solution.md]: We need to update this MSDN link but it appears that the original topic is gone. @ElizabethSamuel-MSFT , is this a topic we can link to for information about adding apps to the app launcher? https://docs.microsoft.com/en-us/office365/admin/manage/customize-the-app-launcher?view=o365-worldwide

Comment (ElizabethSamuel-MSFT) [reply docs/promote-your-office-store-solution.md]: @Lauragra: Perhaps. I can update it to the link you provided but I can't confirm that this is what we should point to.

**Comment by Lauragra** [comment]
OPS Build status updates of commit _[9aa5566](https://github.com/OfficeDev/office-store-docs/pull/193/commits/9aa5566fd98a80fb9d2528f17d2c3d2cda9c1365)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/add-license-checks-to-office-and-sharepoint-add-ins.md](https://github.com/OfficeDev/office-store-docs/blob/elizs-fix-links/docs/add-license-checks-to-office-and-sharepoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/promote-your-office-store-solution.md](https://github.com/OfficeDev/office-store-docs/blob/elizs-fix-links/docs/promote-your-office-store-solution.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C12%5C6275490f-1b1a-267a-353f-0e85484a2ef1%5CPullRequest%5C201903121658523077-193%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BW8s8CEbG3EiNY8yrkHLS4OZTMamanmm%2Fu2AQCx4778%3D&st=2019-03-12T16%3A54%3A49Z&se=2019-04-12T16%3A59%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**